contract_manager/src/evm.ts

```diff
@@ -102,6 +127,12 @@ export class EVMContract extends Contract {
     return Number(result);
   }
 
+  async getBaseUpdateFee() {
+    const pythContract = this.getContract();
+    const result = await pythContract.methods.singleUpdateFeeInWei().call();
+    return { amount: result, denom: "wei" };
```

Comment:
fwiw the `wei` actually means different things on different chains. (e.g., it'll be an amount of matic on polygon.) probably worth noting somewhere.

Reply:
hmmm, in that case, I would make `denom` field optional since it seems it's not generalizing well enough to non-cosmwasm chains.